pkg/costmodel/metrics.go

```diff
@@ -623,6 +636,14 @@ func StartCostModelMetricRecording(a *Accesses) bool {
 					nodeSeen[labelString] = false
 				}
 			}
+			for labelString, seen := range loadBalancerSeen {
+				if !seen {
+					labels := getLabelStringsFromKey(labelString)
+					a.LBCostRecorder.DeleteLabelValues(labels...)
+				} else {
+					loadBalancerSeen[labelString] = false
+				}
+			}
```

Comment:
I just dumb copied this without understanding its purpose because `nodes` and `pvs` also do it.

I assume it's to prevent gauge vector double counting?